source/optional_features/digital_wallets/index.html.md.erb

```diff
@@ -0,0 +1,64 @@
+---
+title: Digital wallets
+weight: 136
+---
+
+# Digital wallets
+
+You can enable [Google Pay](https://pay.google.com) and [Apple Pay](https://www.apple.com/uk/apple-pay/) [external links] to take payments from your users.
+
+You can only enable Google Pay and Apple Pay if you use [Worldpay](/switching_to_live/set_up_a_live_worldpay_account/#set-up-a-live-worldpay-account) as your payment service provider (PSP).
+
+If you use Stripe, [ePDQ](/switching_to_live/set_up_a_live_epdq_account/#set-up-a-live-epdq-account) or [Smartpay](/switching_to_live/set_up_a_live_smartpay_account/#set-up-a-live-smartpay-account) as your PSP, you cannot enable Apple Pay or Google Pay.
```

Comment:
although we could say that Apple/Google Pay will be supported for Stripe soon?

Reply:
Do we know when? Best practice for tech docs is to avoid using imprecise time terms as they lead to more questions / uncertainty. We should either say for example "You will be able to use Apple Pay and Google Pay with Stripe in TIME PERIOD e.g. summer 2019" or leave as is and update when it changes. I recommend keeping as is, and using the suggested content in an announcement or communication to Pay users about the new functionality.

Reply:
do we even need line 12, as 11 explicitly says Google/Apply Pay can only be enabled if the PSP is Worldpay.

Reply:
It's a fair point, we could remove. @katiebates what do you think?

Reply:
fine by me